README.md

```diff
@@ -3,7 +3,7 @@
 [![Build Status](https://travis-ci.org/corrados/jamulus.svg?branch=master)](https://travis-ci.org/corrados/jamulus)
 ![CodeQL](https://github.com/corrados/jamulus/workflows/CodeQL/badge.svg)
 
-Jamulus - Internet Jam Session Software
+Jamulus - Internet Jam Session Software  
```

Comment:
Why is that?

Reply:
To generate new commits and trigger the ci...
Please ignore/ revert that!

Reply:
You should probably ensure that you trigger it differently and open a new PR with only working commits.